packages/docusaurus-theme-classic/src/index.js

```diff
@@ -109,3 +110,76 @@ module.exports = function (context, options) {
     },
   };
 };
+
+const NavbarLinkSchema = yup.object().shape({
+  items: yup
+    .array()
+    .optional()
+    .of(yup.lazy(() => NavbarLinkSchema.default(undefined))),
```

Comment:
not sure to understand what this does

Reply:
I don't know if the navbar is able to have recursive link items (ie more than 1 level). Probably not a big deal if we accept items on >2nd level and if they are simply ignored